packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/ropassword/setup-app.md

```diff
@@ -0,0 +1,8 @@
+4. Select **Native Application** as the **Application type**, then click **Next**.
+5. Specify the **App integration name**.
+6. Select **Resource Owner Password** as one of the allowed **Grant type**.
```

Comment:
```suggestion
6. Select **Resource Owner Password** as the allowed **Grant type**.
```

Reply:
There is another grant type that is automatically selected, the user has to select **Resource Owner Password** in addition to the one that is already there. This wording is from the original Resource owner guide. If this is confusing, I can add the other selected grant type (I was afraid that this selection would change in the near future).